src/colorizer/ColorizeCanvas.ts

```diff
@@ -503,6 +510,7 @@ export default class ColorizeCanvas {
     }
     // Save the array to a texture and pass it into the shader
     this.setUniform("inRangeIds", packDataTexture(Array.from(inRangeIds), FeatureDataType.U8));
+    this.render();
```

Comment:
is this fixing a bug or is it really related to the current changes?

Reply:
It's a bugfix! Before, the canvas thresholding was one update behind thresholding on the UI, but it was less noticeable because we were dealing with continuous numeric values and not categories.